app/src/components/MatricDivider.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import { Box, ResponsiveHidingView, ScreenSize } from '@kibalabs/ui-react';
+
+export const MatricDivider = (): React.ReactElement => {
+  return (
+    <ResponsiveHidingView hiddenBelow={ScreenSize.Medium}>
```

Comment:
i dont think this should be a component (and definitely the hiding should not be in a component). It would be better to just inline this where its used